src/app/settings/sponsored-families.component.html

```diff
@@ -17,16 +17,16 @@ <h1>{{'sponsoredFamilies' | i18n}}</h1>
         </ul>
     </div>
     <form #form (ngSubmit)="submit()" [appApiAction]="formPromise" ngNativeValidate *ngIf="anyOrgsAvailable">
-        <div *ngIf="moreThanOneOrgAvailable" class="form-group col-6">
-            <label for="availableSponsorshipOrg">{{ 'sponsoredFamiliesSelectOffer' | i18n}}</label>
+        <div *ngIf="moreThanOneOrgAvailable" class="form-group col-7">
+            <label for="availableSponsorshipOrg">{{ 'familiesSponsoringOrgSelect' | i18n}}</label>
             <select id="availableSponsorshipOrg" name="Available Sponsorship Organization"
                 [(ngModel)]="selectedSponsorshipOrgId" class="form-control" required>
                 <option value="">-- {{'select' | i18n}} --</option>
                 <option *ngFor="let o of availableSponsorshipOrgs" [ngValue]="o.id">{{o.name}}</option>
             </select>
             <small>{{'sponsoredFamiliesLeaveCopy' | i18n}}</small>
         </div>
-        <div class="form-group col-6">
+        <div class="form-group col-7">
```

Comment:
Why `col-7` here? Most other forms on all the settings pages are in a `col-6`

Reply:
because we want it to be slightly wider to avoid multiple lines in the label